packages/terra-button/src/Button.jsx

```diff
@@ -186,11 +200,16 @@ class Button extends React.Component {
     // Prevent button from showing focus styles when clicked
     this.shouldShowFocus = false;
     // Wait until after onFocus has been triggered on browsers where it will get triggered for click
-    setTimeout(() => { this.shouldShowFocus = true; });
+    setTimeout(() => { this.shouldShowFocus = true; }, 300);
```

Comment:
just curious, does this still work without setting  `300` inside the timeout?

Reply:
handkeKeyDown is called before handleClick above to put focus on the button. Since we don't want focus glow when clicking we need to wait until onclick is triggered before setting shouldShowFocus back to true.